docker-compose.yml

```diff
@@ -140,13 +138,15 @@ services:
   agent:
     image: datadog/agent:latest
     volumes:
-    - /var/run/docker.sock:/var/run/docker.sock:ro
-    - /proc/:/host/proc/:ro
-    - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - /proc/:/host/proc/:ro
+      - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
     environment:
-    - DD_API_KEY=${DATADOG_API_KEY}
-    - DD_APM_ENABLED=true
-    - DD_DOGSTATSD_NON_LOCAL_TRAFFIC=1
+      - DD_API_KEY=${DATADOG_API_KEY}
+      - DD_APM_ENABLED=true
+      - DD_DOGSTATSD_NON_LOCAL_TRAFFIC=1
+    ports:
```

Comment:
I think it is more useful that we forward the real agent port, rather than the agent with a fake api key